azure/table_azure_eventhub_namespace.go

```diff
@@ -0,0 +1,258 @@
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/eventhub/mgmt/2018-01-01-preview/eventhub"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION ////
+
+func tableAzureEventHubNamespace(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_eventhub_namespace",
+		Description: "Azure Event Hub Namespace",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "resource_group"}),
+			Hydrate:           getAzureEventHubNamespace,
+			ShouldIgnoreError: isNotFoundError([]string{"ResourceGroupNotFound", "ResourceNotFound", "404"}),
```

Comment:
@khushboo9024 Can you also validate the error returned, when invalid input is provided?